src/seedu/addressbook/AddressBook.java

```diff
@@ -163,28 +84,34 @@
      * the I/O redirection technique. If not, only the first line of the input
      * text file will be processed.
      */
-    private static final Scanner SCANNER = new Scanner(System.in);
+
     /*
-     * ==============NOTE TO STUDENTS======================================================================
-     * Note that the type of the variable below can also be declared as List<String[]>, as follows:
-     *    private static final List<String[]> ALL_PERSONS = new ArrayList<>()
+     * ==============NOTE TO
+     * STUDENTS=================================================================
+     * ===== Note that the type of the variable below can also be declared as
+     * List<HashMap<PersonProperty, String>>, as follows: private static final
+     * List<HashMap<PersonProperty, String>> ALL_PERSONS = new ArrayList<>()
```

Comment:
Avoid auto-formatting unrelated parts of the code. Doing so can result in unrelated changes being included in the commit. In Eclipse, select the code block you want to auto-format before using the auto-format feature. Otherwise the whole file get auto-formatted. You can use sourcetree to look through changes before committing, and discard any unrelated changes.